Sources/PackageGraph/LocalPackageContainer.swift

```diff
@@ -80,8 +88,14 @@ public final class LocalPackageContainer: PackageContainer {
         toolsVersionLoader: ToolsVersionLoaderProtocol,
         currentToolsVersion: ToolsVersion,
         fileSystem: FileSystem = localFileSystem
-    ) {
-        assert(URL.scheme(package.location) == nil, "unexpected scheme \(URL.scheme(package.location)!) in \(package.location)")
+    ) throws {
+        //assert(URL.scheme(package.location) == nil, "unexpected scheme \(URL.scheme(package.location)!) in \(package.location)")
+        switch package.kind {
+        case .root, .fileSystem:
+            break
+        default:
+            throw InternalError("invalid package type \(package.kind)")
```

Comment:
Same comment as the other message.  If it's already part of the context for whatever handler wraps this call, so it will show up as metadata for a diagnostic, then that's even better.